devfile.yaml

```diff
@@ -0,0 +1,51 @@
+metadata:
+  name: kubernetes-image-puller
+projects:
```

Comment:
we can remove the definition of the project in general e.g. https://github.com/redhat-developer/rh-che/blob/master/devfile.yaml

Reply:
great idea, changed